packages/app/src/components/App.css

```diff
@@ -33,16 +33,15 @@ body {
 
 .app {
     display: grid;
-    font-family: 'roboto';
-    background-color: #aaa;
     grid-gap: 1px;
     grid-template-columns: 250px auto 385px;
-    grid-template-rows: 48px auto auto auto;
     grid-template-areas:
         'headerbar headerbar headerbar'
         'charttype menubar menubar'
         'dimensions chartlayout interpretations'
         'dimensions canvas interpretations';
+    background-color: #aaa;
+    font-family: Roboto, sans-serif;
```

Comment:
Isn't Roboto as the default font inherited from `<UI>`?

Reply:
updated, we didn't set any font on `[dir=ltr,rtl] body`. This has been fixed in `d2/ui` hence removing it.